applets/105-offloading/applet.py

```diff
@@ -0,0 +1,69 @@
+# usage: exec [host] command:"[command]"
+#
+# Check success:
+#  - sudo ethtool -k [device_name]
+#
+# Examples:
+#  - sudo ethtool -K enp4s0 tso off ufo off gso off gro off lro off
+#  - exec-applet 105-offloading alpha command:"tso off ufo off gso off gro off
+#    lro off"
+#
+# Hints:
+#  - "Cannot change [...]" messages probably mean that the value can not be
+#    changed (is fixed)
+
+
+def print_usage(x):
+
+    x.p.msg("\n usage:\n", False)
+    x.p.msg(" - exec 105-offloading [host] command:\"[command]\"\n", False)
+    x.p.msg("\n check success:\n", False)
+    x.p.msg(" - ethtool -k [interface]\n", False)
+
+    return False
```

Comment:
It was used the following way:
if "?" in args:
    return print_usage(x)

Changed it now.